packages/monaco/src/browser/monaco-icon-registry.ts

```diff
@@ -0,0 +1,49 @@
+// *****************************************************************************
+// Copyright (C) 2023 Ericsson and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { injectable } from '@theia/core/shared/inversify';
+import { Event } from '@theia/core';
+import { ThemeIcon } from '@theia/monaco-editor-core/esm/vs/platform/theme/common/themeService';
+import { IconRegistry } from '@theia/core/lib/browser/icon-registry';
+import { IconDefinition, IconFontDefinition, getIconRegistry } from '@theia/monaco-editor-core/esm/vs/platform/theme/common/iconRegistry';
+
+@injectable()
+export class MonacoIconRegistry implements IconRegistry {
+
+    readonly onDidChange: Event<void>;
```

Comment:
Is the "onDidChange" event connected to anything? If not, why do we have it?

Reply:
Thanks for the reviews Thomas,
The event doesn't seem to be connected to anything. Its been removed 👍